sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -210,6 +264,29 @@ void setRegionsContacted(Set<URI> regionsContacted) {
         this.regionsContacted = regionsContacted;
     }
 
+    private void printSystemInformation(StringBuilder stringBuilder) {
+        try {
+            long totalMemory = Runtime.getRuntime().totalMemory() / 1024;
+            long freeMemory = Runtime.getRuntime().freeMemory() / 1024;
+            long maxMemory = Runtime.getRuntime().maxMemory() / 1024;
+            String usedMemory = totalMemory - freeMemory + " KB";
+            String availableMemory = (maxMemory - (totalMemory - freeMemory)) + " KB";
+
+            OperatingSystemMXBean mbean = (com.sun.management.OperatingSystemMXBean)
+                ManagementFactory.getOperatingSystemMXBean();
+            String processCpuLoad = Double.toString(mbean.getProcessCpuLoad());
+            String systemCpuLoad = Double.toString(mbean.getSystemCpuLoad());
```

Comment:
see this comment  from stackoverflow around OperatingSystemMXBean
> Thanks for this answer but i have observed that running this code increases CPU utilization by 30-40%. This means that we get increased CPU Utilization whenever we run this code

from:
https://stackoverflow.com/questions/19781087/using-operatingsystemmxbean-to-get-cpu-usage

Reply:
Thanks that was helpful